app/models/perennial_produce.rb

```diff
@@ -1,6 +1,5 @@
 class PerennialProduce < Produce
-  has_many :Crops
-  validates :name, presence: true, format: { with: /\A[a-zA-Z]*\Z/ } # check if the name has only letters
-  validates :name, :uniqueness => { :case_sensitive => false } # only one produce of a particular name should exist
-  validates :duration, presence: true, format: { with: /\A[0-9]*\Z/ } # duration should only be numbers
+  validates_absence_of :duration
+  validates :start_date, presence: true
+  validates :end_date, presence: true
 end
```

Comment:
Want to make sure that I merged this correctly; should I have kept some of the prior validations?

Reply:
This is fine since PerennialProduce inherites validation from Produce.